com.unity.ml-agents/Tests/Runtime/Unity.ML-Agents.Tests.asmdef

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Unity.ML-Agents.Tests",
+    "references": [
+        "Unity.ML-Agents",
```

Comment:
Is it okay to have `-` in the name? Would `Unity.MLAgents` be better ?

Reply:
I don't think it matters, what do you prefer?